pkg/orchestrator/job-manager.go

```diff
@@ -104,11 +107,20 @@ func (o *Orchestrator) runJob(data *scanData) error {
 // Due to K8s names constraint we will take the image name w/o the tag and repo
 func getSimpleImageName(imageName string) string {
 	repoEnd := strings.LastIndex(imageName, "/")
-	imageName = imageName[repoEnd+1 : len(imageName)]
+	imageName = imageName[repoEnd+1 :]
+
+	digestStart := strings.LastIndex(imageName, "@sha256:")
```

Comment:
From looking at the format [here](https://kubernetes.io/docs/concepts/configuration/overview/#container-images), looks like it's `<image-name>@<digest>` 
Don't know if we should assume sha256